advanced/wallets/react-wallet-v2/src/pages/api/wallet.ts

```diff
@@ -89,7 +89,8 @@ async function handleGetCallsStatus(
   const pimlicoChainName = PIMLICO_NETWORK_NAMES[chain.name]
   const apiKey = process.env.NEXT_PUBLIC_PIMLICO_KEY
   const localBundlerUrl = process.env.NEXT_PUBLIC_LOCAL_BUNDLER_URL
-  const bundlerUrl = localBundlerUrl || `https://api.pimlico.io/v1/${pimlicoChainName}/rpc?apikey=${apiKey}`
+  const bundlerUrl =
+    localBundlerUrl || `https://api.pimlico.io/v1/${pimlicoChainName}/rpc?apikey=${apiKey}`
```

Comment:
Shouldn't we use our [bundler interaction API here](https://specs.walletconnect.com/2.0/specs/servers/blockchain/blockchain-bundler-api)?

Reply:
I will do the cleanup for this. In this PR it just got formatted.